integration-cli/docker_cli_run_unix_test.go

```diff
@@ -270,7 +270,8 @@ func (s *DockerSuite) TestRunOOMExitCode(c *check.C) {
 	errChan := make(chan error)
 	go func() {
 		defer close(errChan)
-		out, exitCode, _ := dockerCmdWithError("run", "-m", "4MB", "busybox", "sh", "-c", "x=a; while true; do x=$x$x$x$x; done")
+		//changes to 10MB from 4MB due to an issue that GCCGO fails to convert properly.
```

Comment:
I think also what @tiborvass might have been asking about is how the conversion isn't working? That bit in the comment could be removed since there's no confirmed reason for the start failure.